src/theory/arith/approx_simplex.cpp

```diff
@@ -2612,11 +2520,16 @@ bool ApproxGLPK::loadRowSumIntoAgg(int nid, int M, const PrimitiveVec& row_sum){
     double coeff = row_sum.coeffs[i];
     ArithVar x = _getArithVar(nid, M, aux_ind);
     if(x == ARITHVAR_SENTINEL){ return true; }
-    Rational c = estimateWithCFE(coeff);
+    Maybe<Rational> c = estimateWithCFE(coeff);
+    if (!c)
+    {
+      return true;
+    }
+
     if(lhs.isKey(x)){
-      lhs.get(x) -= c;
+      lhs.get(x) -= c.value();
     }else{
-      lhs.set(x, -c);
+      lhs.set(x, -c.value());
     }
```

Comment:
Maybe clang-format the whole if-else block?

Reply:
Done